stl/inc/yvals_core.h

```diff
@@ -42,6 +42,8 @@
 //     (partially implemented)
 // P0898R3 Standard Library Concepts
 // P0919R3 Heterogeneous Lookup For Unordered Containers
+// P1227R2 Signed std::ssize(), Unsigned span::size()
+//     (partially implemented)

```

Comment:
I have a mild preference for calling this paper "complete" and treating the remainder as a "patch" to span (since when we implement span we'll be using the WP as the reference and it has this merged into it).

Reply:
If completion here isn't equivalent to completion of the work item and it doesn't indicate that we've implemented the contents of the proposal, then I'm not sure why this list is here and what purpose it serves.

Reply:
All components unique to this proposal are implemented, and so I would mark the work item completed. The only unimplemented parts are patches to other unimplemented proposals.

Reply:
We could spend a lot of time on this and it doesn't really matter (this paper is unusual in that it adds a feature and patches another one; this almost never happens). My preference is to accept this comment as-is and resolve the GitHub issue tracking P1227R2. When we implement `span`, there's no chance of us forgetting this (`span` has churned heavily, so we will definitely use the WP), and at that time we can remove `(partially implemented)` from the comment. Then `yvals_core.h` will correctly indicate that we implemented the patch.

Reply:
Tagging #4 for when someone implements that.